pyee/twisted.py

```diff
@@ -1,16 +1,16 @@
 # -*- coding: utf-8 -*-
 
-from typing import Any, Callable, Dict, Optional, Tuple
+from typing import Any, Callable, Dict, Optional, Self, Tuple
 
 from twisted.internet.defer import Deferred, ensureDeferred
 from twisted.python.failure import Failure
 
 from pyee.base import EventEmitter, PyeeError
 
 try:
-    from asyncio import iscoroutine
+    from asyncio import iscoroutine  # type: ignore
```

Comment:
I might be able to drop this check, as it's here for old versions of Python that may no longer be supported.